### ISSUE TITLE
Rename nextjs

### ISSUE THREAD

**Issue by nihaals** [thunderstore-io/thunderstore-ui#34]
It would be nice to have an explicit name to use when referring to the `nextjs` project. I think the shared components are already fairly clear but referring to `NextJS` might be ambiguous, especially for new contributors. An example of a place this would be used is CI logs. Maybe something creative?

**Comment by with-heart** [comment]
> Maybe something creative?

The first things to pop into my mind were `www` and `site`. How's that for creative!?

**Comment by nihaals** [comment]
I don't think it needs to be something that sets a theme or something (like some companies name all their services relating to food) since we're unlikely to need a huge set of micro-services (at least any time soon). https://github.com/thunderstore-io/Thunderstore/pull/195 is related though so we could follow some kind of convention for the few services we do have

**Comment by MythicManiac** [comment]
I'd intuitively call it `website`, and have probably already called it that in some conversations on Discord

**Comment by nihaals** [comment]
I think `website` it still a bit ambiguous, especially if we're talking generally about Thunderstore. You'd probably end up not using the name and calling it something like "the Next.js frontend"

**Comment by MythicManiac** [comment]
so let's call it nextjs frontend then, eh? 😄

**Comment by nihaals** [comment]
I think if we call it exactly `nextjs-frontend` this works well and is explicit
